.travis.yml

```diff
@@ -1,14 +1,40 @@
-language: node_js
-sudo: false
+language: android
+node_js:
+  - "5.10"
 git:
   depth: 5
 addons:
   sauce_connect: true
+android:
+  components:
+    - build-tools-22.0.1
+    - android-22
+env:
+  - NODE_ENV=cordova
+global:
 before_install:
-  - npm install -g npm@latest
+- echo "Before install stage"
+install:
+- echo "Install stage"
+- sudo apt-get install -y python-software-properties
+- curl --silent --location https://deb.nodesource.com/setup_5.x | sudo bash -
+- sudo apt-get install -y nodejs
+- sudo npm install
+before_script:
+- echo "Before script stage"
 script:
-  - npm run build
-  - npm run test-ci
+- echo "Running tests"
+- npm test
+- echo "Starting build"
+- npm run build
```

Comment:
I think it will be handy to store ci build command sequence in one npm script itself